jme3-effects/src/main/java/com/jme3/post/filters/DepthOfFieldFilter.java

```diff
@@ -243,4 +243,4 @@ public void read(JmeImporter im) throws IOException {
         focusRange = ic.readFloat("focusRange", 10f);
         debugUnfocus = ic.readBoolean("debugUnfocus", false);
     }
-}
+}
```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR